indra/newview/scripts/lua/require/UI.lua

```diff
@@ -150,6 +150,10 @@ function UI.addMenu(...)
     return leap.request('UI', args)
 end
 
+function UI.setMenuVisible(name, visible)
+    return leap.request('UI', {op='setMenuVisible', name=name, visible=visible})
+end
```

Comment:
I'm guessing we don't need to extract a particular item because the only reason for calling `request()` is to handle `"error"`, yes?

Reply:
Yep, similar to `addMenuSeparator`